.github/workflows/deploy-api-oc4.yml

```diff
@@ -0,0 +1,46 @@
+name: Build Efiling Reviewer API Image and Push to Openshift Registry
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - "src/backend/**"
+      - ".github/workflows/deploy-api-oc4.yml"
+
+  # for testing ...
+  pull_request:
+    paths:
+      - "src/backend/**"
+      - ".github/workflows/deploy-api-oc4.yml" 
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
```

Comment:
missing branches here as well

Reply:
"branches" doesn't seem to be a valid option.
I added input: git-ref which defaults to main, but could not find a way yet to enforce only [main]

Reply:
removed workflow-dispatch to avoid this issue completely.